pkgs/tools/security/sn0int/default.nix

```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sn0int";
-  version = "0.24.3";
+  version = "0.25.0";
 
   src = fetchFromGitHub {
     owner = "kpcyrd";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-KREYWM5WOdPzLbOlrATiCCfFwE951KEo03yWNfyG8Bw=";
+    rev = "refs/tags/v${version}";
```

Comment:
Why is the full` refs/tags/` part required?

Reply:
Because we want to prevent branch/tag confusion.